doc/kyuafile.5.in

```diff
@@ -360,6 +392,19 @@ test_suite('FreeBSD')
 plain_test_program{name='the_test',
                    ['custom.FreeBSD-Bug-Id']='category/12345'}
 .Ed
+.Ss FreeBSD jail execution environment
+The following example configures the test to be run within a temporary jail
+created by Kyua with VNET support and the permission to create raw sockets:
+.Bd -literal -offset indent
+syntax(2)
+
+test_suite('FreeBSD')
+
+atf_test_program{name='network_test',
+                 execenv='jail',
+                 execenv_jail='vnet allow.raw_sockets',
+                 required_user='root'}
```

Comment:
_(picking a line)_
The code below doesn't explicitly check to make sure the user has permissions to create jails.
`atf_test_program` allows a config file to incorrectly specify `execenv='jail'` and `required_user='unprivileged_user'`.

Reply:
I was thinking about this and it looks to be simpler to keep the mechanism without additional built-in policy. Who knows, probably in future we will have a capability to create some sort of jails by non-superuser.

For now I propose to improve the documentation (`kyuafile.5`). I've added the key message that root is needed and the examples section was extended with extra explanation. The non-working `jail+unprivileged` combination is mentioned explicitly. What do you think?

Anyway, explicit documentation is a good change here.

Reply:
That sounds great to me!